README.md

```diff
@@ -1 +1,176 @@
+
+- HTML5
+- CSS3
+- Linters
+- Javascript
+- ES6
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.testim.io/blog/what-is-a-linter-heres-a-definition-and-quick-start-guide/">Linters</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Key features of the application.
+
+- **[ES6]**
+- **[Importing-Scripts]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+- To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- An IDE (Intergrated Development Environment ) such as vsode or atom
+- A version control system such as git
+- Node
+
+### Setup
+
+1.  Clone the Repo with:
+
+```
+git clone https://github.com/znation2010/Awesome-Books
+```
+
+2. switch to the cloned directory
+
+```
+cd Awesome-Books
+```
+
+### Installation
+
+1. Install the project with it's depenedcies
+
+```
+npm install
+```
+
+Then open the live server or open the `index.html` file with your browser
+
+### Usage
+
+- this is Awesome Books ES6 project I'll use to publicize the Show.
+
+### Run tests
+
+To run tests, run the following command:
+
+- To run tests run :
+- npx hint .
+- npx stylelint "\*_/_.{css,scss}"
+- npx eslint .
+
+<!-- Deployment -->
+
+### Deployment
+
+- Project is deployed using [GitHub Pages](https://pages.github.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Zerihun Abune**
+
+- GitHub: (https://github.com/znation2010)
+- Twitter: (https://twitter.com/zerihunabune)
+- LinkedIn: (https://linkedin.com/in/zerihun-abune-4bb11b196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features to be added:
+
+- [ ] **[Project: Accessibility and Contacts]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the <a href="https://github.com/znation2010/Awesome-Books/issues"> issues page </a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd like to acknowledge and appreciate my coding and learning partner for their input in the project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
```

Comment:
- [ ] Good job adding the MIT file, The problem is you need to update its content in regard to your name and project title. Please replace the `[YOUR NAME]` part with your name, `[APP TYPE]` with your app type, and the [YEAR] with the current year.